recipes/base_install.rb

```diff
@@ -116,6 +116,7 @@
   mode '0644'
 end
 
+node.default['nfs']['threads'] = node['cfncluster']['nfs_server_threads'].to_i
```

Comment:
If we do revisit this, we need to do this configuration at runtime, not at AMI build time.